net/netclient/patches/010-install-initd-fix.patch

```diff
@@ -0,0 +1,59 @@
+--- a/daemon/initd_linux.go
++++ b/daemon/initd_linux.go
```

Comment:
WTF is this? 

Formality reasons:
1. No commit subject
2. No commit description
3. Who is the author?

Didnt you somewhere complain that there are patches without Git header and that you are doing it like it should?

Anyway:

I wonder, what is the reason to have it here as downstream patch. This is not fix, this is hack or maybe even workaround, but for workaround, there is a lot of work...

Reply:
Commit description addresses your first question, which should have been phrased better, but I'll reply this one time.

As per https://openwrt.org/docs/guide-developer/toolchain/use-patches-with-buildsystem, the requirements you listed  are only valid for "target/linux/generic and <arch>:".

Please update that documentation page with information on how to generate the patch containing commit subject, commit description and SoB if you want me to update the patch.

Reply:
Then please look at https://openwrt.org/submitting-patches , there is what are you looking for.

Reply:
You need to see it in bigger picture, why it is required. The doc, which I mention, says you do need it like this, but it explains partially why. It is not something like harrasment, abusive nor that you can feel that I am toxic. 

The reason, why it is necessary to have Git header is this:
https://github.com/openwrt/openwrt/commit/eac6fe2d70cacbddb736e814674dfa070faac0a8

There is everything what are you looking even in the referenced pull request. Hopefully that explains, why I could be so strict. I Just dont want that the situation with OpenWrt specific patches without any details happen again. In Turris, we were dealing with various issues and looking at those hack patches was one of our job and we were like why? When we were using vanilla kernel, there were no issues and that aha moment came to our mind... hack patches, carried here for ages and the developer, Who created is unreachable and no one wants to open the can with worms.

Reply:
Right, maybe **you** should read the content at the links you've posted. Patch conforms to the requirements.

Reply:
It does not even comply with patch checklist (the first link), the Git header still missing (the second link), so... it does not. :)